spec/amp-story-player.md

```diff
@@ -126,3 +130,192 @@ CSS variable with the URL pointing to the poster image of the story.
   </a>
 </amp-story-player>
 ```
+
+## Programmatic Control
+
+Call the player's various methods to programmatically control the player. These methods are exposed on the DOM HTML element, `const playerEl = document.querySelector('amp-story-player')` and on the global class variable,`const player = new AmpStoryPlayer(window, playerEl)`.
+
+### Methods
+
+#### load
+
+Will initialize the player manually. This can be useful when the player is dynamically added such as when using frameworks like React.
+
+```
+function StoryPlayer({ url, title, poster, width = 360, height = 600 }) {
+
+       useEffect(() => {
+           const playerEl = document.body.querySelector('amp-story-player');
+           const player = new AmpStoryPlayer(window, playerEl);
+           player.load();
+	}, []);
+
+	return (
+		<div>
+		  <amp-story-player style={{ width: `${width}px`, height: `${height}px`}}>
+			<a href={url}>
+			    {title}
+			</a>
+		  </amp-story-player>
+		</div>
+	);
+}
+```
+
+#### go
+
+**Parameters**
+
+- number: the story in the player to which you want to move, relative to the current story.
+
+If the player is currently on the third story out of ten stories, `player.go(1)` will go forward one story to the fourth story and `player.go(-1)` will go backward one story to the second story. If no value is passed or if delta equals 0, current story will persist and no action will be taken.
```

Comment:
Please split these into two examples, when written sequentially they're getting confusing: go(1) goes to the 4th story and go(-1) should go back to the third. I know what you meant but we should remove any ambiguity.